charts/fluid-datamigrate/juicefs/templates/statefulset.yaml

```diff
@@ -0,0 +1,76 @@
+{{- if gt (.Values.datamigrate.parallelism | int) 1 }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ printf "%s-workers" .Release.Name }}
+  labels:
+    release: {{ .Release.Name }}
+    role: datamigrate-job
+  ownerReferences:
+  {{- if .Values.owner.enabled }}
+    - apiVersion: {{ .Values.owner.apiVersion }}
+      blockOwnerDeletion: {{ .Values.owner.blockOwnerDeletion }}
+      controller: {{ .Values.owner.controller }}
+      kind: {{ .Values.owner.kind }}
+      name: {{ .Values.owner.name }}
+      uid: {{ .Values.owner.uid }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ printf "%s-workers" .Release.Name }}
+  # match the service name
+  serviceName: {{ printf "%s-svc" .Release.Name }}
+  replicas: {{ .Values.datamigrate.parallelism }}
+  podManagementPolicy: Parallel
+  template:
+    metadata:
+      labels:
+        app: {{ printf "%s-workers" .Release.Name }}
+    spec:
+      containers:
+        - name: worker
```

Comment:
Do we now have a way for users to define the sts's Pod's resources?

Reply:
@TrafalgarZZZ  the new field is a struct `corev1.ResourceRequirements`, can not define in the `DataMigrate.spec.options` . So uses a new field called `workerResources` in the `DataMigrate.spec` ? how about add a new struct ParallelismOptions for parallel migrate and move all newly added field to it, it make sense only when `Datamigrate.spec.Parallelism` > 1 ?

Reply:
> As there may have another sts related configuration requirements in the future, how about add a new struct `ParallelismOptions` for parallel migrate and move all newly added field（Parallelism，SSHSecretName，SSHPort，WorkersReadyTimeout） to it ?

`ParallelismOptions` LGTM.